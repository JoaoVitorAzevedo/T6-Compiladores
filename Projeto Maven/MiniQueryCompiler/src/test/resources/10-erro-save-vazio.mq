LOAD "data.json"
SAVE AS ""