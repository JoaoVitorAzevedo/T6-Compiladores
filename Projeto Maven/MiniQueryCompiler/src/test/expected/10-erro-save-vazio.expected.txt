ERROS SEMÂNTICOS:
ERRO (Linha 2): Nome do arquivo de saída não pode ser vazio